src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -1162,7 +1180,7 @@ internal ImageClassificationEstimator(IHostEnvironment env, Options options, Dnn
             _host = Contracts.CheckRef(env, nameof(env)).Register(nameof(ImageClassificationEstimator));
             _options = options;
             _dnnModel = dnnModel;
-            _tfInputTypes = new[] { TF_DataType.TF_STRING };
+            _inputTypes = new[] { new VectorDataViewType(NumberDataViewType.Byte) };
```

Comment:
>_inputTypes [](start = 12, length = 11)

This field is not needed. In `GetOutputSchema` you can just compare the input column item type directly to `NumberDataViewType.Byte`.
But I was wondering - should this constructor perhaps perform a check that the input type of the TensorFlow graph node is also byte? And same for the label? Or even that these inputs exist in the graph?